packages/flutter/test/widgets/live_text_input_test_util.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Maybe rename this file to "live_text_utils.dart", similar to https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/editable_text_utils.dart?